x-pack/plugins/apm/public/components/app/service_overview/latency_chart/index.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiSelect, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useCallback, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { LatencyAggregationType } from '../../../../../common/latency_aggregation_types';
+import { useUrlParams } from '../../../../context/url_params_context/use_url_params';
+import { FETCH_STATUS } from '../../../../hooks/use_fetcher';
+import { useTransactionChartsFetcher } from '../../../../hooks/use_transaction_charts_fetcher';
+import { TimeseriesChart } from '../../../shared/charts/timeseries_chart';
+import { getResponseTimeTickFormatter } from '../../../shared/charts/transaction_charts/helper';
+import { useFormatter } from '../../../shared/charts/transaction_charts/use_formatter';
+import * as urlHelpers from '../../../shared/Links/url_helpers';
+
+interface Props {
+  height?: number;
+}
+
+const options: Array<{ value: LatencyAggregationType; text: string }> = [
+  { value: 'avg', text: 'Average' },
+  { value: 'p95', text: '95th percentile' },
+  { value: 'p99', text: '99th percentile' },
+];
+
+export function LatencyChart({ height }: Props) {
+  const history = useHistory();
+  const { urlParams } = useUrlParams();
+  const { latencyAggregationType } = urlParams;
+
+  // const {
+  //   transactionChartsData,
+  //   transactionChartsStatus,
+  // } = useTransactionChartsFetcher();
+  // const { responseTimeSeries, anomalySeries } = transactionChartsData;
+
+  // const timeSeries = responseTimeSeries
+  //   ? [responseTimeSeries[selectedOption]]
+  //   : [];
+  // const { formatter } = useFormatter(timeSeries);
+  // const options = responseTimeSeries?.map(({ title }, index) => ({
+  //   value: index,
+  //   text: title,
+  // }));
```

Comment:
Did you commit this by a mistake?

Reply:
I haven't finished it yet... Will be back to fix some stuff I left behind to refactor the API as we discussed.